tutorial/thermodynamics/Harmonic_Approx.ipynb

```diff
@@ -0,0 +1,1065 @@
+{
```

Comment:
This is great. Would you mind also adding an example/thermodynamics.ipynb file where you just show the functioning of the functions? Basically this tutorial minus the text.